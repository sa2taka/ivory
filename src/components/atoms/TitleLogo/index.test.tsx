import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { TitleLogo } from './index';

describe('TitleLogo', () => {
  test('is rendered', () => {
    const wrapper = shallow(<TitleLogo />);
    expect(wrapper).toBeTruthy();
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
