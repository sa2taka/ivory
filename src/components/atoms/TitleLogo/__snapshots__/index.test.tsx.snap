// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleLogo is rendered 1`] = `
<div
  className="flex items-center undefined"
>
  <img
    className="h-8 w-8"
    src="ivory-logo.png"
  />
  <span
    className="ml-2 text-2xl font-semibold"
  >
    Ivory
  </span>
</div>
`;
